src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,65 @@
+package Duke;
+
+/**
+ * Creates a Task object.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor
+     *
+     * @param description name of task
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Marks a task as done.
+     */
+    public void setAsDone() {
+        isDone = true;
+    }
+
+    /**
+     * Returns a cross if task is not done.
+     * Returns a tick if task is done.
+     *
+     * @return tick or cross symbol
+     */
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
```

Comment:
You can consider saving the symbols as a constant variable, this makes the code easier to read at first glance and easier to edit if needed